src/devtoolsPanel/AuraInspectorTransactionPanel.js

```diff
@@ -166,6 +174,27 @@ export default function AuraInspectorTransactionPanel(devtoolsPanel) {
         }
     }
 
+    function FilterText_OnChange(event) {
+        var text = event.srcElement;
+
+        transactionView.setFilter(text.value);
+    }
+
+    function debounce(func, wait, immediate) {
```

Comment:
These should be compiled now and work with npm.

Any reason you didn't const debounce = require('debounce');  ?

Reply:
I try to see how some similar was done on the other views and they recreate each time this function.

Moreover, into the package.json, there is no direct declaration of this module (into dependencies and even into devDependencies. In our case, only dependencies is interesting).

That's mean this module was fetched as a chjld dependency and this is so not recommended to import it (because an update of the parent dependency could remove it). 
I could if you want import (to version 1.1.0 as we can see into the lock files) and use it into the transaction panel

Regards